meson/README.md

```diff
@@ -0,0 +1,17 @@
+# meson
+
+This package provides the meson build system binaries
+
+## Usage
+
+Typically this is a build dependency that can be added to your
+plan.sh:
+
+    pkg_build_deps=(core/meson core/python)
+
+To be able to use it, you need to add meson's python library to `PYTHONPATH` like so:
```

Comment:
Can we use the new runtime environment variables to handle this for versions that support that: https://github.com/habitat-sh/core-plans/pull/998/files